public/pages/Dashboard/Container/DashboardOverview.tsx

```diff
@@ -94,8 +93,7 @@ export function DashboardOverview() {
   const [selectedDetectorStates, setSelectedDetectorStates] = useState(
     [] as string[]
   );
```

Comment:
Small thing, but one suggestion I have would be making selectedDetectorStates a DETECTOR_STATE[] rather than string[] for consistency. You would probably have to change your handleDetectorStateFilterChange() like this: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/development/public/pages/DetectorsList/List/List.tsx#L188. Thoughts?

Reply:
good point. Changing to `DETECTOR_STATE []`